src/main/java/com/flightstats/hub/webhook/WebhookService.java

```diff
@@ -70,6 +73,13 @@ public WebhookService(@Named("Webhook") Dao<Webhook> webhookDao, WebhookValidato
         return preExisting;
     }
 
+    public Optional<Webhook> upsertTagWebhook(Webhook webhook, Optional<Webhook> preExisting) {
+        webhookDao.upsert(webhook);
+//        webhookManager.notifyWatchers(webhook);
```

Comment:
If we are updating a tag webhook, does that need to be processed by webhookManager?